### PR TITLE
Infer-target

### PR DESCRIPTION
**Pull request by jorana**
**What is the problem / what does the code in this PR do**
Online_peak_monitor was removed from the target list because of the str match with 'peak'. Fix that.

While at it, also add some more logging statements and go for a little simpler syntax.

Honestly, this is not the prettiest part of the code. Will do some work on it when we add some new infer-logic for the linked mode as well as NV/MV processing.

### CODE REVIEW

File: bin/bootstrax [hunk 1]
```diff
@@ -538,11 +544,19 @@ def infer_target(rd):
 
     n_fails = rd['bootstrax'].get('n_failures', 0)
     if n_fails:
-        log.debug(f'override_target::\tdeleting targets')
+        log.debug(f'Deleting targets')
         for _rm_t, _rm_t_after in remove_target_after_fails.items():
-            _t = [__t for __t in _t if (_rm_t not in __t and n_fails < _rm_t_after)]
-            _pp = [__pp for __pp in _pp if (_rm_t not in __pp and n_fails < _rm_t_after)]
+            for __i, __t in enumerate(_t):
```
Comment (darrylmasson) [inline bin/bootstrax:549]: I think it's worth cleaning up the variable names here, `i` is cleaner than `__i`, and `potential_targets` is cleaner than `_t`. This isn't the c++ standard library, I don't think we need _this_ many underscores.

File: bin/bootstrax [hunk 2]
```diff
@@ -538,11 +544,19 @@ def infer_target(rd):
...
+                if __t in _rm_t and n_fails > _rm_t_after:
```
Comment (darrylmasson) [inline bin/bootstrax:550]: Is the logic here the right way around? Previously this was `_rm_t in __t`

**Review by darrylmasson — COMMENTED** [review]
One `in` seems to me to be reversed

File: bin/bootstrax (hunk 1 above)
Comment (jorana) [reply bin/bootstrax:549]: >Honestly, this is not the prettiest part of the code. Will do some work on it when we add some new infer-logic for the linked mode as well as NV/MV processing.

yeah, this whole thing got messier than it supposed to

File: bin/bootstrax (hunk 2 above)
Comment (jorana) [reply bin/bootstrax:550]: Logic is reverted, now this `if` -> deletes targets if we want to, whereas it used to say `if` -> keep the target. Therefore there is also no `not`  involved for `__t in _rm_t`

**Pull request merged**